### planner environment settings ###############################################

# the size of the used hash map (the bigger the better); should be something 
# with 2^X (initially 2^16=65536)
max_hash_size: 65536

# the heuristic that should be used to estimate the step costs of a planning state
# possible choices: EuclideanHeuristic, EuclStepCostHeuristic, PathCostHeuristic
heuristic_type: EuclideanHeuristic

accuracy:
  # footstep collision check accuracy
  # - 0 (just the circumcircle of the foot)
  # - 1 (just the incircle of the foot)
  # - 2 (circumcircle and incircle recursivly checked for the whole foot)
  collision_check: 2
  cell_size: 0.01
  num_angle_bins: 72
  
  
### planner settings ###########################################################

# define the planner of choice provided by the SBPL package
# - ARAPlanner
# - ADPlanner
planner_type: ARAPlanner

# search until a specific time limit is reached or first solution is found 
search_until_first_solution: False
# the maximum search time if search_until_first_solution is set to False
allocated_time: 7.0
initial_epsilon: 8.0
forward_search: False
